docs/src/main/java/org/geotools/cql/ECQLExamples.java

```diff
@@ -115,7 +116,12 @@ public static void main(String[] args) {
                 case 17: 
                 	utcTimeZone();
```

Comment:
Pre-existing tabs here could be expanded.

Reply:
The file is full of them... I would need a reformat

Reply:
Your discretion. I often give in to temptation and fix formatting adjacent to changes, but this is not required, and mass reformatting is not in general desirable.